docs/reference/profileservice.rst

```diff
@@ -41,7 +41,7 @@ IsActiveContext
 Models the request to determine is the user is currently allowed to obtain tokens. It contains these properties:
 
 ``Subject``
-    The ``ClaimsPrincipal`` modeling the user. If the request The claims from the user's cooke will be in the ``ClaimsPrincipal``.
+    The ``ClaimsPrincipal`` modeling the user. If the request [I don't pretend to know what you meant to say about the request! (But you didn't say it... this fragment is followed by a new sentence or perhaps the "then" that follows an incomplete "if" --> ] The claims from the user's cookie will be in the ``ClaimsPrincipal``.
```

Comment:
the annotation in brackets should be removed

Reply:
Agreed -- but what we really need is the corrected text desired. I will remove my comment, but that leaves the two spots in need of correction by someone else. That make sense?

Reply:
Probably a better way to handle that one, but I'm a novice on git. :)